test/hotspot/jtreg/testlibrary/ctw/src/sun/hotspot/tools/ctw/CtwRunner.java

```diff
@@ -303,6 +303,9 @@ private String[] cmd(long classStart, long classStop) {
                 "-XX:+StressCCP",
                 "-XX:+StressMacroExpansion",
                 "-XX:+StressIncrementalInlining",
+                "-XX:+StressBailout",
+                "-XX:StressBailoutProbability=100_000_000",
```

Comment:
Hmm, does this very high mean-to-bailout imply that we rarely bailout? I guess it is a delicate line between bailing out too much and then all the compilations do not complete, and not bailing out enough, and not testing the bailout in combination with the stress flags enough. How do you know that this value is a good choice?